tests/suites/test_suite_pk.function

```diff
@@ -106,6 +106,27 @@ mbedtls_svc_key_id_t pk_psa_genkey( void )
     psa_set_key_bits( &attributes, bits );
     PSA_ASSERT( psa_generate_key( &attributes, &key ) );
 
+exit:
+    return( key );
+}
+
+/*
+ * Generate an RSA key using PSA and return the key identifier of that key,
+ * or 0 if the key generation failed.
+ */
+mbedtls_svc_key_id_t pk_psa_genkey_rsa( void )
+{
+    mbedtls_svc_key_id_t key;
+    psa_key_attributes_t attributes = PSA_KEY_ATTRIBUTES_INIT;
+    const psa_key_type_t type = PSA_KEY_TYPE_RSA_KEY_PAIR;
+    const size_t bits = 256;
```

Comment:
I think even for tests it's better to use somewhat realistic sizes. 256 is so small that some driver implementations might reject it outright, and tests should work with drivers if possible. While 2048 is the most common size, 1024 should be enough to be accepted by all implementations (it's still used in some places).